doctr/io/elements.py

```diff
@@ -12,6 +12,11 @@
 from doctr.utils.common_types import BoundingBox, RotatedBbox
 from doctr.utils.repr import NestedObject
 
+import doctr
+from xml.dom import minidom
```

Comment:
you're not using this import (cf. https://github.com/mindee/doctr/pull/544/checks?check_run_id=4067302369)